GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1533,10 +1632,18 @@ - (void)OAuthLoginWithAddScopesFlow:(BOOL)addScopesFlow
                                        hint:_hint
                            additionalScopes:nil
                                       nonce:nonce
+                                tokenClaims:tokenClaims
                                  completion:completion];
       }
     }
 
+    if (tokenClaimsError) {
+      XCTestExpectation *tokenClaimsErrorExpectation =
+          [self expectationWithDescription:@"Callback called"];
```

Comment:
Let's not reuse "Callback called" since it's extremely vague. Please use something more descriptive like "Token claims error expectation."